### PR TITLE
[API 1016] Tech Debt: Find a better way to merge PDF's together

### PR DESCRIPTION
**Pull request by templeman15**
## Description of change
This PR removes dependency from combine_pdf gem by concatenating PDFs using pdftk in the `HigherLevelReviewPdfConstructor`.

## Original issue(s)
[Tech Debt: Find a better way to merge PDF's together](https://vajira.max.gov/browse/API-1016)

### CODE REVIEW

File: modules/appeals_api/spec/workers/higher_level_review_pdf_submit_job_spec.rb [hunk 1]
```diff
@@ -98,16 +101,60 @@
...
   private
 
+  # Converts PDF at path to array of md5's, one md5 for each page of the document.
+  # Problem: Taking an md5 of the PDF itself causes a mis-match between the generated and expected PDFs due to
+  # differences in metadata.
+  #
+  # @param [String] path The path to the PDF
+  # @param [String] identifier The unique identifier of the higher level review
+  # @return [[String]] Array of md5's
+  def pdf_to_md5s(path, identifier)
```
Comment (templeman15) [inline modules/appeals_api/spec/workers/higher_level_review_pdf_submit_job_spec.rb:124]: @bastosmichael do we have a good place to put spec helper methods that can be shared across modules or should we have the duplicate code in this case of these two methods: `pdf_to_md5s` and `pdf_to_image`?

Comment (bastosmichael) [reply modules/appeals_api/spec/workers/higher_level_review_pdf_submit_job_spec.rb:124]: sounds good.

File: modules/claims_api/spec/workers/poa_form_builder_job_spec.rb [hunk 2]
```diff
@@ -74,5 +74,63 @@
...
+  private
+
+  # Converts PDF at path to array of md5's, one md5 for each page of the document.
+  # Problem: Taking an md5 of the PDF itself causes a mis-match between the generated and expected PDFs due to
+  # differences in metadata.
+  #
+  # @param [String] path The path to the PDF
+  # @param [String] identifier The unique identifier of the power of attorney
+  # @return [[String]] Array of md5's
+  def pdf_to_md5s(path, identifier)
```
Comment (bastosmichael) [inline modules/claims_api/spec/workers/poa_form_builder_job_spec.rb:103]: Can we consolidate this into some sort of spec helper for others to use?

File: modules/claims_api/spec/workers/poa_form_builder_job_spec.rb [hunk 3]
```diff
@@ -74,5 +74,63 @@
...
+      File.delete(image_path) if File.exist?(image_path)
+      md5
+    end
+  end
+
+  # Converts PDF at path to jpg's
+  # @param [String] path The path to the PDF
+  # @param [String] identifier The unique identifier of the power of attorney
+  # @return [[String]] Array of image_paths
+  def pdf_to_image(path, identifier)
```
Comment (bastosmichael) [inline modules/claims_api/spec/workers/poa_form_builder_job_spec.rb:117]: Move to spec helper for others to reuse.

**Review by bastosmichael — CHANGES_REQUESTED** [review]
Consolidating those new methods into a spec helper will help others utilize it in the future.

File: modules/claims_api/spec/workers/poa_form_builder_job_spec.rb (hunk 2 above)
Comment (templeman15) [reply modules/claims_api/spec/workers/poa_form_builder_job_spec.rb]: Updated!

File: modules/claims_api/spec/workers/poa_form_builder_job_spec.rb (hunk 3 above)
Comment (templeman15) [reply modules/claims_api/spec/workers/poa_form_builder_job_spec.rb]: Updated!

**Pull request closed without merging**